frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -246,7 +246,7 @@ function Pages(): JSX.Element {
                         title="Instance settings"
                         icon={<DatabaseOutlined />}
```

Comment:
Let's not use AntD icons here for consistency, all `LemonButton`s use Material Icons. I replaced it with the "Dns" one: https://fonts.google.com/icons?icon.query=system